electron-builder.json

```diff
@@ -77,7 +77,13 @@
       }
     ],
     "hardenedRuntime": true,
-    "gatekeeperAssess": true
+    "gatekeeperAssess": true,
+    "entitlements": "./build/entitlements.mac.plist",
+    "entitlementsInherit": "./build/entitlements.mac.plist",
+    "extendInfo": {
+      "NSMicrophoneUsageDescription": "Third party plugins (like jitsi) will use the microphone to provide in-app videoconference",
```

Comment:
Microphone access may be used by Mattermost plugins, such as Jitsi video conferencing.